pkg/columns/formatter/textcolumns/textcolumns.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package textcolumns
```

Comment:
It is named `textcolumns`, but its main function is `NewFormatter`? I would suggest `formatter` as the package name, don't you think?

Reply:
There's also a csv folder (and later on possibly a json folder) with formatters. So I thought csv.NewFormatter() (actually, right now it's csv.NewCSVFormatter() - should rename this to NewFormatter() as well), json.NewFormatter(), textcolumns.NewFormatter() would make sense.